CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/model/UiModel.kt

```diff
@@ -0,0 +1,13 @@
+package co.nimblehq.coroutine.model
+
+import co.nimblehq.coroutine.domain.model.Model
+
+data class UiModel(
+    val id: Int
+)
+
+private fun Model.toUiModel(): UiModel {
+    return UiModel(
+        id = id ?: -1
+    )
+}
```

Comment:
Ditto here on the shortening the code idea 😁
```suggestion
private fun Model.toUiModel(): UiModel = UiModel(id = id ?: -1)
```